lib/solidus_subscriptions/configuration.rb

```diff
@@ -80,5 +80,9 @@ def subscribable_class
     def churn_buster?
       churn_buster_account_id.present? && churn_buster_api_key.present?
     end
+
+    def process_job_error_handler
+      @process_job_error_handler ||= nil
+    end
```

Comment:
This isn't required since the attribute will already be `nil` if it's not set. You can just move the attribute to the `attr_accessor` call instead and remove this method.